### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by amityadav0**
- Bug fix: we were using test functions in src, that's why build folder had two folders (src and test)
and we were not able to import directly

### CODE REVIEW

File: packages/algorand-js/tsconfig.json [hunk 1]
```diff
@@ -1,10 +0,0 @@
-{
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "outDir": "./build"
-  },
-  "include": [
-    "src",
-    "../../@types"
-  ]
-}
```
Comment (ratik21) [inline packages/algorand-js/tsconfig.json:10]: What was different before? It wasn't a symlink?

File: examples/crowdfunding/test/crowdfundingTest.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {
   stringToBytes,
   TransactionType
 } from '@algorand-builder/algob';
-import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js/build/src/index';
+import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js';
 import { assert } from 'chai';
 
 import { getAcc } from './common';
```
Comment (ratik21) [inline examples/crowdfunding/test/crowdfundingTest.js:12]: I noticed we're using `import {..}` here in `.js`, whereas in other files we are using `const {..} = require('..');`. Is there any specific reason for doing that?

File: packages/algorand-js/tsconfig.json (hunk 1 above)
Comment (amityadav0) [reply packages/algorand-js/tsconfig.json:10]: yes

File: examples/crowdfunding/test/crowdfundingTest.js (hunk 2 above)
Comment (amityadav0) [reply examples/crowdfunding/test/crowdfundingTest.js:12]: no, not any specific reason

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**